### DISCUSSION TITLE
Spotlight nesting

### DISCUSSION THREAD

**Discussion by lukasbash** [mantinedev/mantine#5151] [Feature requests]
What do you think about the possibility to nest spotlights or spotlight actions?
I am thinking about a use case to allow the user to access any part of the application through spotlight, similar to Alfred (the macOS native spotlight alternative).
Imagine a lot of entries which could be grouped (i.e. not the currently available action groups) in some semantic way.
Example: Think about an entry "Navigate to...", when the user presses enter, and the actions get replaced/reduced to related action children.

Why not use action groups and render several hundred action items at once? Imagine offering a lot of actions under a specific term A. The user has the options to navigate to A, create A, search A, etc. and the results would be too fuzzy.

Just an idea, What do you think?

**Comment by rtivital** [comment]
It is planned to add this feature, but it is at the bottom of the backlog

**Reply by wagnergerald** [reply]
This feature would be really awesome 🚀
